InputUser.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { TextInput, StyleSheet, View } from "react-native";
+
+export const InputUser = () => {
+  const [text, onChangeText] = React.useState();
+  const [description, onChangeTextDescription] = React.useState();
+  return (
+    <View style={styles.container}>
+      <TextInput
+        style={styles.input}
+        onChangeText={onChangeText}
+        value={text}
```

Comment:
Mm, both values are `text`. Do you think this is the more representative name for this value?